stdlib/REPL/src/latex_symbols.jl

```diff
@@ -119,6 +119,46 @@ const latex_symbols = Dict(
     "\\euler" => "ℯ",
     "\\ohm" => "Ω",
 
+    # Music Symbols
+    # Music Symbols - Accidentals
+    "\\flatflat" => "𝄫",
+    "\\sharpsharp" => "𝄪",
+    # Music Symbols - Codas
+    "\\leftrepeatsign" => "𝄆",
+    "\\rightrepeatsign" => "𝄇",
+    "\\dalsegno" => "𝄉",
+    "\\dacapo" => "𝄊",
+    "\\segno" => "𝄋",
+    "\\coda" => "𝄌",
+    # Music Symbols - Clefs
+    "\\clefG" => "𝄞",
+    "\\clefG8va" => "𝄟",
+    "\\clefG8vb" => "𝄠",
+    "\\clefC" => "𝄡",
+    "\\clefF" => "𝄢",
+    "\\clefF8va" => "𝄣",
+    "\\clefF8vb" => "𝄤",
+     # Music Symbols - Rests
+    "\\restMulti" => "𝄺",
+    "\\restWhole" => "𝄻",
+    "\\restHalf" => "𝄼",
+    "\\restQuarter" => "𝄽",
+    "\\rest8th" => "𝄾",
+    "\\rest16th" => "𝄿",
+    "\\rest32th" => "𝅀",
+    "\\rest64th" => "𝅁",
+    "\\rest128th" => "𝅂",
+    # Music Symbols - Notes
+    "\\noteDoubleWhole" => "𝅜",
```

Comment:
I don't think most of the other symbols use camelCase?

Reply:
Thank you, I have changed this.